.github/workflows/ci.yml

```diff
@@ -16,48 +16,36 @@ jobs:
       fail-fast: false
       matrix:
         runs-on:
+          - ubuntu-20.04
           - ubuntu-22.04
-          # - ubuntu-24.04 # FIXME: Re-enable
-          # - macos-13
+          - ubuntu-24.04
+          - macos-13
           - macos-14
-          # - macos-15 # FIXME: Re-enable
-          # - windows-2019 # FIXME: Re-enable
+          - macos-15
+          - windows-2019
           - windows-2022
-          # - windows-2025 # FIXME: Re-enable
+          - windows-2025
         compiler:
           - gcc-13
           - gcc-14
           - clang-19.1.0
         exclude:
-          # FIXME: Fix these exclusions if possible, and re-include them.
+          - runs-on: ubuntu-20.04
+            compiler: gcc-14 # Cannot be installed.
           - runs-on: ubuntu-22.04
             compiler: gcc-14 # Cannot be installed.
           - runs-on: macos-13
-            compiler: gcc-13 # Fails at: /Users/runner/work/tirex-tracker/tirex-tracker/c/build/_deps/cpuinfo-src/src/api.c:6
+            compiler: gcc-13 # Known issue with gcc and the GitHub macOS 13 runner
```

Comment:
Would be nice to link to the corresponding issue if it is a known ~feature~bug 🙂